lib/types/library/LibraryBuilder.js

```diff
@@ -67,6 +67,16 @@ class LibraryBuilder extends AbstractBuilder {
 			});
 		});
 
+		this.addTask("generateManifestBundle", () => {
+			const generateManifestBundle = tasks.generateManifestBundle;
+			return generateManifestBundle({
+				workspace: resourceCollections.workspace,
+				options: {
+					namespace: project.metadata.namespace
```

Comment:
Library projects do not have a `metadata.namespace` property. Reason is that they might have multiple namespaces (like sap.ui.core).

I think this still works as the namespace option is only used to:
> `// Root-project only: Remove namespace prefix if given`

See:
https://github.com/SAP/ui5-builder/blob/bb2fcf20526c6072278f9346c97bb184bb23b82b/lib/processors/bundlers/manifestBundler.js#L104-L106

But even if it works, we should make the parameter optional in the JSDoc and do not pass it here as it is always undefined.

Reply:
Actually I think this doesn't work at all. The second parameter of `addBuffer` is the path *inside* the zip file, which must not start with `/`. Have you tested this by any chance?

I'll have a look and try and crate a test for this.

Reply:
Your points have been tackled with the new upload of the commit. I also merged the commits and aligned to the commit message rules.